Neos.Flow/Classes/Security/AccountInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
```

Comment:
Please add a strict_type declaration to all introduced files (https://discuss.neos.io/t/usage-of-php-strict-typing-by-default/3861)

Reply:
I don't know how to mark this is as done so it's not marked as "blocking"?